src/CodeGen_C.cpp

```diff
@@ -68,8 +68,9 @@ const string globals =
     "}\n"
     "\n"
 
-    // TODO: this next chunk is copy-pasted from posix_math.cpp. A
-    // better solution for the C runtime would be nice.
+    // We now add definitions of things in the runtime which are
+    // intended to be inlined into every module. The redundancy is
+    // regrettable (FIXME).
```

Comment:
We should probably (at least) open an Issue on this if we really want to fix it eventually.